tests/training/helpers.py

```diff
@@ -0,0 +1,37 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+import shutil
+from pathlib import Path
+
+from argilla.training import ArgillaTrainer
+
+
+def cleanup(trainer: ArgillaTrainer, output_dir: str, train: bool = True):
```

Comment:
I've just found a potential alternative to this:
https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture

I must say that I couldn't find this earlier when I googled...
This approach is fine too, though.